crates/matrix-sdk-common/src/linked_chunk/updates.rs

```diff
@@ -242,6 +247,19 @@ impl<Item, Gap> UpdatesInner<Item, Gap> {
         }
     }
 
+    /// Clear all pending updates.
+    fn clear(&mut self) {
```

Comment:
Would it maybe be better if we call this `clear_pending`?

Reply:
Why not!